lib/srv/reexec.go

```diff
@@ -138,6 +165,24 @@ func RunCommand() (io.Writer, int, error) {
 			return errorWriter, teleport.RemoteCommandFailure, trace.BadParameter("pty and tty not found")
 		}
 		errorWriter = tty
+		ttyName, err = os.Readlink(tty.Name())
+		if err != nil {
+			return errorWriter, teleport.RemoteCommandFailure, trace.BadParameter("Failed to resolve tty soft link: %v", err)
```

Comment:
nit: I don't believe error message start with a Capital letter. These should be lowe-cased to make them composable - at least it's how it is elsewhere in the code.

Reply:
hmm, RJ just said they should be capitalized

Reply:
Nevermind, that was another error, adjusted.